### PR TITLE
Feat/135 temp change news section to faq previews

### PR DESCRIPTION
**Pull request by mikolaj-jalocha**

### CODE REVIEW

**Review by simon-the-shark — COMMENTED** [review]
Looks good. We'll need to see how it looks with the final data there (cause now it's a bit biggy :P) Maybe we'll need to decrease the size a bit. But let's wait for now for the final data. One note left. Also maybe you can delete the previous `newsRepository` for now (I think we had one).

File: lib/features/guide_view/repository/guide_view_repository.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+typedef GuidePosts = Query$GetGuidePosts$FAQ_Types;
+typedef _GetGuidePosts = WatchOptions$Query$GetGuidePosts;
+
+@riverpod
+Stream<List<GuidePosts?>?> guidePostsRepository(
+  GuidePostsRepositoryRef ref,
+) async* {
+  final stream = ref.watchQueryWithCache(
+    _GetGuidePosts(eagerlyFetchResults: true),
+    TtlKey.guideDetailsRepository,
```
[the remark this answers is not in the document]
Comment (mikolaj-jalocha) [reply lib/features/guide_view/repository/guide_view_repository.dart]: Of course.. I wanted to change it later (after repo setup) but short term memory.....

**Comment by mikolaj-jalocha** [comment]
> Looks good. We'll need to see how it looks with the final data there (cause now it's a bit biggy :P) Maybe we'll need to decrease the size a bit. But let's wait for now for the final data. One note left. Also maybe you can delete the previous `newsRepository` for now (I think we had one).

Thanks for review. All changes applied. 
Totally agree with long card issue :-) It'd need a fix probably.

**Review by simon-the-shark — APPROVED** [review]
LGTM! 😎🦈😎

**Pull request merged**